unit/models/customer.py

```diff
@@ -150,3 +152,24 @@ def to_dict(self) -> Dict:
             parameters["sort"] = self.sort
         return parameters
 
+
+class ArchiveCustomerRequest(UnitRequest):
+    def __init__(self, customer_id: str, reason: Optional[ArchiveReason] = None):
```

Comment:
reason is Optional, so I made it to be equal to None by default and users won't need to pass this param if there is no need